docker-compose.yml

```diff
@@ -280,7 +280,7 @@ services:
     image: docker.io/permitio/pdp-v2:0.2.37
     restart: always
     environment:
-      - PDP_CONTROL_PLANE=${PERMIT_API}
+      - PDP_CONTROL_PLANE=https://api.eu-central-1.permit.io
```

Comment:
nit: I would still keep this behind an env variable. It is not necessarily a secret, but it doesn't need to be hardcoded either.